pallets/messages/src/mock.rs

```diff
@@ -156,11 +156,14 @@ impl AccountProvider for AccountHandler {
 	}
 
 	fn ensure_valid_schema_grant(
-		_provider: Provider,
-		_delegator: Delegator,
+		provider: Provider,
+		delegator: Delegator,
 		_schema_id: SchemaId,
 	) -> DispatchResult {
-		Ok(())
+		match Self::get_provider_info_of(delegator, provider) {
```

Comment:
This might break unit test just to be sure

Reply:
This might break the benchmarks if it returns error especially for messages pallet, same for other changes there was a runtime-benchmark version and not runtime-benchmark versio

Reply:
It did. Pushed a fix. @saraswatpuneet